mathesar/filters.py

```diff
@@ -3,6 +3,16 @@
 
 from mathesar.models import Schema, Table, Database
 
+FILTER_OPTIONS_BY_TYPE_NAME = {
+    'Boolean':
```

Comment:
Please use the `MathesarTypeIdentifier` enum from `mathesar/database/types.py` to get the identifier here so we don't depend on a string that could change.